Frontend/ui-common/src/Components/PopUpModal/View.purs

```diff
@@ -587,3 +657,10 @@ tipsViewConfig state = let
   }
   in tipsViewConfig'
 
+
+parseConerRadius :: Corners -> Number
+parseConerRadius corners =
```

Comment:
@aayushagarwal2001 can you rename this to parseCornerRadius. 
 And why are we not assigning the corner radius sent directly here?
 If we need only the radius then radius alone as number.